storage/migrations/12_analysis_node_height.up.sql

```diff
@@ -0,0 +1,14 @@
+BEGIN;
+
+-- Tracks the current (consensus) height of the node. TODO: expand to paratime heights.
+CREATE TABLE chain.node_heights
+(
+  analyzer TEXT NOT NULL PRIMARY KEY,
```

Comment:
I think it makes more sense this being `layer` instead of `analyzer`? As there can be multiple consensus related (or runtime related) analyzers, but the latest node_height depends on the layer.

Reply:
Good point; updated!